### PR TITLE
[Feat] Added layout

### PR DESCRIPTION
**Pull request by aryan-13**
Made the following changes: 
- Basic Layout of the app
- added icons

Preview: 
image

### CODE REVIEW

File: src/Components/MainSection/TaskList/TaskList.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.main-section-container {
+	padding: 2rem 1rem;
+	margin: 1.5rem;
+	border-radius: 1rem;
+	background-color: lightpink;
```
Comment (anshikaa12) [inline src/Components/MainSection/TaskList/TaskList.css:5]: you can use variables for color naming

File: src/Components/Timer/Timer.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.timer-container {
+	padding: 2rem 1rem;
+	margin: 1.5rem;
+	border-radius: 1rem;
+	background-color: orange;
```
Comment (anshikaa12) [inline src/Components/Timer/Timer.css:5]: same suggestion

File: public/index.html [hunk 3]
```diff
@@ -24,12 +24,16 @@
...
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+		<script
+			src="https://kit.fontawesome.com/9b9e4cb61d.js"
+			crossorigin="anonymous"
+		></script>
+		<title>React App</title>
```
Comment (junaidshaikh-js) [inline public/index.html]: Provide good title name for your website

**Review by anshikaa12 — APPROVED** [review]
clean and understandable code
well structured
i liked the UI

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	--color-grey-dark-2: #999;
+	--color-grey-dark-3: #333;
+
+	--color-white: #fff;
+	--color-black: #2f2e41;
+}
+
+* {
+	box-sizing: border-box;
+}
```
Comment (junaidshaikh-js) [inline src/App.css]: Can be moved in the top most rule set

**Review by junaidshaikh-js — COMMENTED** [review]
Good work! 

Suggested two changes, everything else looks good and can be merged.

File: public/index.html (hunk 3 above)
Comment (aryan-13) [reply public/index.html]: Yes, Changed it

File: src/App.css (hunk 4 above)
Comment (aryan-13) [reply src/App.css]: Done

File: src/Components/MainSection/TaskList/TaskList.css (hunk 1 above)
Comment (aryan-13) [reply src/Components/MainSection/TaskList/TaskList.css:5]: These are just for visualising the layout, I'll add variables later

**Pull request merged**